### DISCUSSION TITLE
Could it be possible to have the connector record pre-upgraded and engineered suits and weapons from concourses?

### DISCUSSION THREAD

**Discussion by SourceOneDocs** [EDCD/EDMarketConnector#2298] [Ideas]
It would be fantastic if a commander checking out the stock in Pioneer stores resulted in recording what was there and automatically populating the Sharing is Caring forum thread or just show it in the EdMarketConnector app itself.

**Comment by Rixxan** [comment]
That sounds like something a plug-in would be best designed to do. 

Well I'm sure there would be users for such a feature, we try and keep EDMC pretty clean for users to expand how they want. I don't think this would be something we'd want to adopt as part of the core program.
